TF-14957 Add Team Tokens Index API (#667)
hashicorp/terraform-docs-common: website/docs/cloud-docs/users-teams-organizations/api-tokens.mdx

API tokens may belong to a specific team. Team API tokens allow access to the workspaces that the team has access to, without being tied to any specific user.

> To manage the API token for a team, go to **Organization settings > API Tokens > Team Token tab** to create a new token or search for a team that you would like to manage the API token for.

Reviewer: For brevity, we can shorten this to:
```suggestion
Navigate to the **Organization settings > API Tokens > Team Token** tab to manage API tokens for a team or create new team tokens.
```
